internal/setup/setup.go

```diff
@@ -40,6 +40,10 @@ type Package struct {
 	Slices  map[string]*Slice
 }
 
+func (p *Package) MarshalYAML() (interface{}, error) {
+	return packageToYAML(p)
```

Comment:
Why not inline the `*ToYAML` functions?

Reply:
I can do that for `packageToYAML`, but not for `sliceToYAML` though since I am using that one in a few other places which requires me to return a `*yamlSlice` type instead of `interface{}`. I mean, I can but then I have to get the `*yamlSlice` type manually from the interface. Hence, the different functions.

Reply:
Then keep it like this, thanks!